core/templates/core/base_international.html

```diff
@@ -52,15 +52,14 @@
 
         <div id="header-bar">
             <div class="container">
-                <p class="government-links"><img src="{% static 'images/international/df-it-186-digi-aw-white.svg' %}" alt="Department for International Trade" height="20">
-                </p>
+                <p class="government-links"></p>
```

Comment:
(comment for line 36) would it make more sense for the p tag here to be an h2 instead? since it has a subheading on the domestic page version - so we could have an h2 and then a p tag there https://app.zeplin.io/project/58ac6a5b11c1d45f91a7d47c/screen/5a0461d2e0f018aafd15c2b5